### PR TITLE
Complete 4 - GET /api/articles'

### PR DESCRIPTION
**Pull request by waiwong-nc**
Complete  GET /api/articles 
Test with correct return and descending order with jest-sorted.

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -40,5 +40,38 @@ describe('API',() => {
...
+              expect(article).toEqual(
+                expect.objectContaining({
+                  author: expect.any(String),
+                  title: expect.any(String),
+                  article_id: expect.any(Number),
+                  topic: expect.any(String),
+                  created_at: expect.any(String),
+                  votes: expect.any(Number),
+                  comment_count: expect.toBeOneOf([null, expect.any(String)]),
+                })
```
Comment (Jodiech) [inline __tests__/app.test.js:61]: This shouldn't default to null. It should be zero. It shouldn't be a string but a number

File: models/articles.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const db = require("../db/connection");
+
+exports.selectAllArticles = () => {
+  const sql =
```
Comment (Jodiech) [inline models/articles.js]: This query is wrong based on what it is returning in your test. Please see the previous comment and amend 👍

**Review by Jodiech — COMMENTED** [review]
Please amend your test to make sure comment_count is returning the right value. Good work Wilson!

**Review by k-huggs — APPROVED** [review]
All good to merge

**Pull request merged**